### PR TITLE
ANTTASK-52 Update to SQ Runner 2.5

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: src/main/java/org/sonarsource/scanner/ant/SonarQubeTask.java [hunk 1]
```diff
@@ -48,13 +57,37 @@ public void execute() {
...
       .unmask("org.sonar.ant")
-      .setApp("Ant", SonarQubeTaskUtils.getTaskVersion())
-      .addExtensions(getProject())
-      .execute();
+      .setApp("Ant", SonarQubeTaskUtils.getTaskVersion());
+
+    runner.start();
+    try {
+      runner.addExtensions(getProject());
+    } catch (Exception e) {
```
Comment (SonarTech) [inline src/main/java/org/sonarsource/scanner/ant/SonarQubeTask.java]: CRITICAL Either log or rethrow this exception. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1166)

File: src/main/java/org/sonarsource/scanner/ant/SonarQubeTask.java [hunk 2]
```diff
@@ -48,13 +57,37 @@ public void execute() {
...
+      // Not supported in recent SQ versions. Ignore
+    }
+    runner.runAnalysis(properties);
+    runner.stop();
+
   }
 
+  private static int toAntLevel(Level level) {
```
Comment (SonarTech) [inline src/main/java/org/sonarsource/scanner/ant/SonarQubeTask.java]: MAJOR The Cyclomatic Complexity of this method "toAntLevel" is 12 which is greater than 10 authorized. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 1 critical
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 1 critical
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by dbmeneses** [comment]
LGTM

**Pull request merged**